query-connector/design-review/README.MD

```diff
@@ -1,22 +1,22 @@
 # Design Review Script
 
-A bash script intended to allow designers and members of the team to easily spin up an instance of the TEFCA Viewer on their local machines for UI/UX review. Currently only MacOS is supported.
+A bash script intended to allow designers and members of the team to easily spin up an instance of the Query Connector on their local machines for UI/UX review. Currently only MacOS is supported.
 
 ## Initial Setup
 
 In order to use this script a copy must be downloaded onto your computer from GitHub . To do this follow these instructions.
+
 1. Open the Terminal application.
-2. Copy and paste `curl https://raw.githubusercontent.com/CDCgov/phdi/main/containers/tefca-viewer/design-review/design-review.sh -O` into the Terminal prompt and press enter. This command uses the wget program to download a copy the `design-review.sh` file from this directory to the root level of your user directory e.g. `Users/johndoe`.
+2. Copy and paste `curl https://raw.githubusercontent.com/CDCgov/dibbs-query-connector/main/query-connector/design-review/design-review.sh -O` into the Terminal prompt and press enter. This command uses the wget program to download a copy the `design-review.sh` file from this directory to the root level of your user directory e.g. `Users/johndoe`.
```

Comment:
@fzhao99 , this script works for me if I copy it directly. what I think is happening is when I run this command currently:
`curl https://raw.githubusercontent.com/CDCgov/dibbs-query-connector/main/query-connector/design-review/design-review.sh -O`

it is copying the version of the script from `main`, which will still be pointing to the old repo stuff. So for testing purposes, @mikang should copy the version of the script here: https://github.com/CDCgov/dibbs-query-connector/blob/91a8f60915c3e39d47c1849f498dbc4e60a079cc/query-connector/design-review/design-review.sh.

and overwrite her existing `design-review.sh`. then run steps 3 and 4.